modules/tools.py

```diff
@@ -179,7 +187,7 @@ async def run_bash_state(
 }
 
 
-async def set_timeout(_state: State, timeout: str | int) -> str:
+async def set_timeout(_state: State, timeout: str | int | Any) -> str:
```

Comment:
Not blocking, but putting the `Any` here makes me wonder if we should be handling this differently, on the actor instead for example. But probably not worth looking into this now.